docs/en_US/index.rst

```diff
@@ -0,0 +1,565 @@
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <!-- Installation -->
+  <div class="gap">
+    <h2 class="title">Installation</h2>
+    <div>
+      <h3 class="second-title">Install</h3>
+      <div class="gap2">
+        NNI supports and is tested on Ubuntu >= 16.04, macOS >= 10.14.1,
+        and Windows 10 >= 1809. Simply run the following <code>pip install</code>
+        in an environment that has <code>python 64-bit >= 3.6</code>.
+      </div>
+      <div class="command-intro">Linux or macOS</div>
+      <div class="command">python3 -m pip install --upgrade nni</div>
+      <div class="command-intro">Windows</div>
+      <div class="command">python -m pip install --upgrade nni</div>
+      <div class="command-intro">If you want to try latest code, please <a href="installation.html">install
+          NNI</a> from source code.
+      </div>
+      <div class="chinese">For detail system requirements of NNI, please refer to <a href="Tutorial/InstallationLinux.html">here</a>
+        for Linux & macOS, and <a href="Tutorial/InstallationWin.html">here</a> for Windows.</div>
+    </div>
+    <div>
+      <p>Note:</p>
+      <ul>
+        <li>If there is any privilege issue, add --user to install NNI in the user directory.</li>
+        <li class="rowHeight">Currently NNI on Windows supports local, remote and pai mode. Anaconda or Miniconda is highly
+          recommended to install <a href="Tutorial/InstallationWin.html">NNI on Windows</a>.</li>
+        <li>If there is any error like Segmentation fault, please refer to <a
+            href="installation.html">FAQ</a>. For FAQ on Windows, please refer
+          to <a href="Tutorial/InstallationWin.html">NNI on Windows</a>.</li>
+      </ul>
+    </div>
+    <div>
+      <h3 class="second-title gap">Verify installation</h3>
+      <div>
+        The following example is built on TensorFlow 1.x. Make sure <b>TensorFlow 1.x is used</b> when running
+        it.
+      </div>
+      <ul>
+        <li>
+          <div class="command-intro">Download the examples via clone the source code.</div>
+          <div class="command">git clone -b v2.6 https://github.com/Microsoft/nni.git</div>
+        </li>
+        <li>
+          <div>Run the MNIST example.</div>
+          <div class="command-intro">Linux or macOS</div>
+          <div class="command">nnictl create --config nni/examples/trials/mnist-pytorch/config.yml</div>
+          <div class="command-intro">Windows</div>
+          <div class="command">nnictl create --config nni\examples\trials\mnist-pytorch\config_windows.yml</div>
+        </li>
+        <li>
+          <div class="rowHeight">
+            Wait for the message INFO: Successfully started experiment! in the command line.
+            This message indicates that your experiment has been successfully started.
+            You can explore the experiment using the Web UI url.
+          </div>
+          <!-- Indentation affects style！ -->
+          <pre class="code">
```

Comment:
class name is `code`, but why not delete it in css?

Reply:
because it has some side effects. Let me change its name.